boards/xtensa/xt-sim/xt-sim.dts

```diff
@@ -0,0 +1,13 @@
+/dts-v1/;
+
+#include "sample_controller.dtsi"
+
+/ {
+	model = "xt-sim";
+	compatible = "sample,controller";
```

Comment:
Hm... I don't do the cadence tools so much, but I'm pretty sure that "xt-sim" is **not** a "sample_controller".  Those are two different CPU instances (actually xt-sim is a whole family I think).